openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -33,6 +33,15 @@
   box-shadow: none;
 }
 
+.deleted-button {
+  margin-left: 4px;
+  padding: 2px 8px;
+  font-weight: 500;
+  border-radius: @border-radius-base;
+  color: @error-color;
+  background-color: @error-light-color;
+}
```

Comment:
dont we have a style for the deleted button? please check existing implementation and style for the same